Sources/Turf/GeoJSON.swift

```diff
@@ -121,10 +120,12 @@ public struct Feature: Codable {
     }
 }
 
-public enum GeoJSONType: String {
+public enum GeoJSONType: String, CaseIterable {
```

Comment:
I suggest to use `CaseIterable` instead of manually defining `allValues`

Reply:
Yes, `CaseIterable` is much cleaner but it wasn't available until Swift 5.0, which explains why the Linux CI bot on Swift 4.1 fails. Feel free to bump the requirement to 5+ in the SPM manifest.